src/app/app.component.css

```diff
@@ -1,3 +1,4 @@
 .container {
   font-family: Arial;
+  min-height: 90vh;
```

Comment:
It will work on some screens, but we need solution that will work on every screen

Reply:
Updated the code to use a fixed position footer